content/docs/next-release/operator.md

```diff
@@ -881,6 +881,44 @@ kubectl delete jaeger simplest
 Deleting the instance will not remove the data from any permanent storage used with this instance. Data from in-memory instances, however, will be lost.
 {{< /info >}}
 
+# Tracing the operator
+
+Starting from version 1.16.0, the Jaeger Operator is able to generate spans related to its own operations. To take advantage of that, the `operator.yaml` has to be configured to enable tracing by setting the flag `--tracing-enabled=true` to the `args` of the container and to add a Jaeger Agent as sidecar to the pod. Here's an excerpt from an `operator.yaml` that has tracing enabled:
+
+```yaml
+...
+        # .Spec.Template.Spec.Containers[0].Args
+        args: ["start", "--tracing-enabled=true"]
+...
+      # add as a second container to .Spec.Template.Spec.Containers
+      - name: jaeger-agent
+        image: jaegertracing/jaeger-agent:1.15.1
```

Comment:
Maybe `latest` instead of `1.15.1`? as this will be included in the `1.16` docs.